pom.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.exist-db</groupId>
+        <artifactId>exist-apps-parent</artifactId>
+        <version>1.3</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.exist-db</groupId>
+    <artifactId>atomic-wiki</artifactId>
```

Comment:
this app now installs to `apps/atomic-wiki/` the old app to `apps/wiki/` so a lot of links are now broken. See `/apps/atomic-wiki/data/WikiIntro.html` it no longer has styling for probably the same reason